projects/forms/package.json

```diff
@@ -7,6 +7,7 @@
     "@angular/forms": ">12.2.15"
   },
   "dependencies": {
+    "luxon": "^3.0.1",
```

Comment:
Q: Why luxon?

Naar mijn weten word er momenteel voornamelijk date-fns als alternatief op momentjs gebruikt. We zouden eigenlijk tot een consensus moeten komen binnen SHD welke date package we zouden willen gebruiken (niet alleen voor deze repo).

Reply:
Eigenlijk puur omdat ik geen momentjs wou gebruiken en had gehoord dat luxon "all het goeie van momentjs maar treeshakable" is. 

Maar inderdaad zeker gelijk dat we daar een consensus in moeten hebben, ik wil het gerust naar `date-fns` veranderen, ik heb niet meteen een sterke mening daarin.

Reply:
Date-fns is ook tree-shakeable en vergeleken met luxon heeft het meer features. Wij gebruiken (op de meeste projecten) vooral date-fns, dus ik zou dat gewoon gelijk trekken zodat we niet met 30 verschillende date packages zitten :)